app/src/main/java/org/sopt/at/SignUpActivity.kt

```diff
@@ -0,0 +1,334 @@
+                Color.Red
+            } else {
+                Color.Gray
+            },
+            fontSize = 12.sp
+        )
+        Spacer(modifier = Modifier.weight(5f))
+        Button(
+            onClick = onNext,
+            modifier = Modifier
+                .fillMaxWidth()
+                .clip(RoundedCornerShape(2.dp))
+                .height(52.dp),
+            shape = RoundedCornerShape(2.dp),
+            enabled = text != "",
+            colors = ButtonDefaults.buttonColors(
+                disabledContainerColor = Color.Black,
+                disabledContentColor = Color.White,
+                containerColor = Color.White,
+                contentColor = Color.Black,
+            ),
+            border = BorderStroke(0.5.dp, Color.Gray)
+        ) {
+            Text("다음", fontSize = 14.sp)
+        }
+        Spacer(modifier = Modifier.weight(1f))
+    }
+}
+
+@Composable
+fun PasswordInputView(
+    text: String,
+    onValueChange: (String) -> Unit,
+    onSignUp: () -> Unit,
+    isError: Boolean
+) {
+    var isVisible by remember { mutableStateOf(false) }
+    Column(
+        modifier = Modifier
+            .width(440.dp)
+            .padding(horizontal = 4.dp)
+    ) {
+        Text(
+            "비밀번호를 입력해주세요.",
+            fontSize = 24.sp,
+            fontWeight = FontWeight.Bold,
+            modifier = Modifier
+                .padding(bottom = 36.dp),
+            color = Color.White
+        )
+        OutlinedTextField(
+            value = text,
+            onValueChange = onValueChange,
+            placeholder = { Text("비밀번호", color = Color.LightGray) },
+            singleLine = true,
+            modifier = Modifier
+                .fillMaxWidth()
+                .clip(RoundedCornerShape(2.dp))
+                .background(color = Color.DarkGray)
+                .height(52.dp),
+            colors = OutlinedTextFieldDefaults.colors(
+                focusedBorderColor = Color.LightGray,
+                unfocusedBorderColor = Color.Transparent,
+                cursorColor = Color.White,
+            ),
+            shape = RoundedCornerShape(2.dp),
+            textStyle = TextStyle(color = Color.White),
+            trailingIcon =
+                {
+                    IconButton(onClick = { isVisible = !isVisible }) {
+                        Icon(
+                            painter = painterResource(
+                                if (isVisible) R.drawable.baseline_remove_red_eye_24
+                                else R.drawable.baseline_visibility_off_24
+                            ),
+                            contentDescription = "",
+                            tint = Color.LightGray
+                        )
+                    }
+
+                },
+            visualTransformation = if (!isVisible) {
+                PasswordVisualTransformation()
+            } else {
+                VisualTransformation.None
+            }
+        )
```

Comment:
```suggestion
            visualTransformation = if (!isVisible) PasswordVisualTransformation() else VisualTransformation.None
```
이렇게 쓰면 코드 더 깔끔하게 쓸 수 있어요 ~!